### ISSUE TITLE
Issue building new updater -- Unable to open booster.bin

### ISSUE THREAD

**Issue by tcal-x** [im-tomu/foboot#187]
Please let me know if I missed a step.   This is what I ran:

```
cd hw
python3 foboot-bitstream.py --revision pvt --seed 19

cd ../booster
cc -O2 -o make-booster -I ./include make-booster.c

cd ../releases
bash ./release.sh pvt
```

With the last step, I get this error:

```
+++ dirname ./release.sh
++ cd ./..
++ pwd
+ root=/home/tim/fomu/foboot
+ platform=pvt
+ input=/home/tim/fomu/foboot/hw/build
+ '[' -z pvt ']'
+ '[' pvt = evt ']'
+ '[' pvt = evt-spi ']'
+ '[' pvt = pvt ']'
+ spi_id=0xc2152815
+ cd /home/tim/fomu/foboot
++ git describe --tags --dirty=+
+ release=v2.0.3-53-gaf4d063+
+ output=/home/tim/fomu/foboot/releases/v2.0.3-53-gaf4d063+
+ mkdir -p /home/tim/fomu/foboot/releases/v2.0.3-53-gaf4d063+
+ cp /home/tim/fomu/foboot/hw/build/gateware/top.bin /home/tim/fomu/foboot/releases/v2.0.3-53-gaf4d063+/pvt-foboot-v2.0.3-53-gaf4d063+.dfu
+ cp /home/tim/fomu/foboot/hw/build/gateware/top-multiboot.bin /home/tim/fomu/foboot/releases/v2.0.3-53-gaf4d063+/pvt-multiboot-v2.0.3-53-gaf4d063+.bin
+ cp /home/tim/fomu/foboot/hw/build/software/bios/bios.elf /home/tim/fomu/foboot/releases/v2.0.3-53-gaf4d063+/pvt-bios-v2.0.3-53-gaf4d063+.elf
+ cp /home/tim/fomu/foboot/hw/build/software/include/generated/csr.h /home/tim/fomu/foboot/releases/v2.0.3-53-gaf4d063+/pvt-csr-v2.0.3-53-gaf4d063+.h
+ cp /home/tim/fomu/foboot/hw/build/software/include/generated/soc.h /home/tim/fomu/foboot/releases/v2.0.3-53-gaf4d063+/pvt-soc-v2.0.3-53-gaf4d063+.h
+ cd /home/tim/fomu/foboot/booster
+ ./make-booster 0xc2152815 /home/tim/fomu/foboot/hw/build/gateware/top-multiboot.bin /home/tim/fomu/foboot/releases/v2.0.3-53-gaf4d063+/pvt-updater-v2.0.3-53-gaf4d063+.dfu
Unable to open booster.bin (try running 'make'): No such file or directory
```

**Comment by xobs** [comment]
Booster needs to be built first. In the `booster` directory, type `make` to build it.

Note that you should make sure your copy of Foboot works correctly before flashing it, because there's no easy way to roll back without a test jig.

**Comment by tcal-x** [comment]
> Booster needs to be built first. In the `booster` directory, type `make` to build it.

Thanks @xobs , that is what I was missing.  I'll send a PR adding it to the instructions.

**Comment by tcal-x** [comment]
Fixed via PR #188 .
